### PR TITLE
Multiple destinations for trips and an install script

### PR DESCRIPTION
**Pull request by evancohen**
Users can now pass in multiple trips into the config. This is a BREAKING CHANGE.

@shoaibuddin sent a PR (#202) which got me thinking about ways the installation process could be easier. I reworked the script he wrote to be a bit more robust + user friendly. The script (which is a one-off) takes care of:
- installing node (if you don't have it already)
- rotating the screen (requires reboot)
- removing the mouse when inactive
- cloning the repo
- installing dependencies

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                                                                
+ _____ ______   ___  ________  ________  ________  ________     
+|\   _ \  _   \|\  \|\   __  \|\   __  \|\   __  \|\   __  \    
+\ \  \\\__\ \  \ \  \ \  \|\  \ \  \|\  \ \  \|\  \ \  \|\  \   
+ \ \  \\|__| \  \ \  \ \   _  _\ \   _  _\ \  \\\  \ \   _  _\  
+  \ \  \    \ \  \ \  \ \  \\  \\ \  \\  \\ \  \\\  \ \  \\  \  
+   \ \__\    \ \__\ \__\ \__\\ _\\ \__\\ _\\ \_______\ \__\\ _\ 
+    \|__|     \|__|\|__|\|__|\|__|\|__|\|__|\|_______|\|__|\|__|
+
+EOF
```
Comment (javiersigler) [inline scripts/install.sh]: Which tool do u use to generate this? It looks awesome

Comment (shoaibuddin) [inline scripts/install.sh]: @javiersigler http://textart.io/

Comment (evancohen) [inline scripts/install.sh]: It's a site l that I've been using for ~8 years:
http://patorjk.com/software/taag

It never goes out of style!

Comment (javiersigler) [inline scripts/install.sh]: @evancohen @shoaibuddin thank you guys. I've always wanted to know it

File: scripts/install.sh [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+printf "This script will install the smart-mirror and it's dependencies.\n"
+
+# Ensure the use would like to start the install
```
Comment (ashishsc) [inline scripts/install.sh]: User*

File: scripts/install.sh [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+\
+#Rotate the display (smart-mirror)\
+display_rotate=1' /boot/config.txt
+fi
+
+printf "\nChecking for node...\n"
+node --version | grep ${NODE_VERSION}
+if [[ $? != 0 ]] ;
+then
+    # Install Node
```
Comment (ashishsc) [inline scripts/install.sh]: Consider using [n](https://github.com/tj/n)  for easy install

**Pull request merged**